Rakefile

```diff
@@ -6,4 +6,4 @@ require File.expand_path("config/application", __dir__)
 Rails.application.load_tasks
 
 Rake::Task[:default].clear unless Rails.env.production?
-task default: %i[lint spec]
+task default: %i[lint spec jasmine:ci]
```

Comment:
We should also expand the lint task to include the JS linting, example: https://github.com/alphagov/content-publisher/blob/d41fabb5ce00f137ed6b409c130c51c499f91b9f/lib/tasks/lint.rake#L6